src/mmw/package.json

```diff
@@ -28,6 +29,7 @@
     "jstify": "0.9.0",
     "leaflet": "0.7.3",
     "leaflet-draw": "0.2.3",
+    "livereloadify": "2.0.0",
```

Comment:
Should this be added to `devDependencies` so that browserify doesn't bundle it? Or is that taken care of separately?